src/notification/audio.rs

```diff
@@ -0,0 +1,67 @@
+use std::io::Cursor;
+
+use data::audio;
+use kira::{
+    manager::{
+        backend::{cpal::CpalBackend, DefaultBackend},
+        AudioManager, AudioManagerSettings,
+    },
+    sound::static_sound::StaticSoundData,
+};
+
+pub enum State {
+    Ready { manager: AudioManager<CpalBackend> },
+    Unsupported,
+}
+
+impl State {
+    pub fn new() -> Self {
+        let Ok(manager) = AudioManager::<DefaultBackend>::new(AudioManagerSettings::default())
+        else {
+            return Self::Unsupported;
+        };
+
+        State::Ready { manager }
+    }
+
+    pub fn play(&mut self, sound: &audio::Sound) -> Result<(), PlayError> {
+        if let State::Ready { manager } = self {
+            let data = match sound {
+                audio::Sound::Internal(internal) => {
+                    let bytes = internal.bytes();
+                    StaticSoundData::from_cursor(Cursor::new(bytes))?
+                }
+                audio::Sound::External(sound_name) => {
+                    let Some(sound_path) = data::audio::find_external_sound(sound_name) else {
+                        return Err(PlayError::NoSoundFound);
+                    };
+
+                    StaticSoundData::from_file(sound_path)?
```

Comment:
We're doing blocking IO here during the main thread / `update`.

We either need to:

- load sound data into memory after we parse the config. We can hook this via Config::load like we do with themes and password files
- fire Tasks to load the data then play it once loaded. This is a bit more complex.

I vote for the first option